### DISCUSSION TITLE
Suggestions for chat titles

### DISCUSSION THREAD

**Discussion by OrO-c** [Taewan-P/gpt_mobile#99] [Ideas]
![Screenshot_20241102175925.jpg](https://github.com/user-attachments/assets/2f3e785c-3384-47a6-8dc2-706bdcbaa734)

The chat title of this software is the first message sent, and it is suggested that the model summarize the conversation content and generate the title

**Comment by Taewan-P** [comment]
For the first step, I am planning on implementing chat title change feature first. I will try summarization feature after that.

**Reply by OrO-c** [reply]
ok

**Comment by Taewan-P** [comment]
This is now implemented in #117.
